src/main/java/uk/antiperson/stackmob/tasks/MergeTask.java

```diff
@@ -19,6 +19,23 @@ public MergeTask(StackMob sm) {
     }
 
     private void checkEntity(StackEntity original, boolean checkHasMoved, double checkHasMovedDistance) {
+        if(original.getEntityConfig().isMsptReactiveEnabled()){
```

Comment:
Is it likely that this would need to be enabled/disabled on a per entity type basis? Because at the moment this is doing the check for each entity, when it could just be checked once at the beginning of the task.